src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1767,11 +1865,15 @@ bool entitiesQuery
       // FIXME P4: Once issue #1705 is implemented, this check can be removed.
       if (qStringFilters(sco->value, filters) != true)
       {
-        if (badInputP)
+        if (badInputP)  // Bad Input reported by higher level
```

Comment:
Within that "up to each user" context that was a suggestion :)

I mean, for me is more clear to have `if (badInpuP != NULL)`. In general in my opinion, `if (foo)` should be used only if `foo` is a boolean.

It is a matter of personal taste of course. If you prefer stay as it is now, then NTC this thread comment.